src/Components/Employees/Form/index.js

```diff
@@ -20,54 +31,50 @@ const Form = () => {
     password: ''
   });
 
-  useEffect(async () => {
-    try {
-      if (id) {
-        const response = await fetch(`${process.env.REACT_APP_API_URL}/employees/${id}`, {
-          method: 'GET'
-        });
-        const data = await response.json();
-        setIsEditing(true);
-        setEmployeeInput({
-          name: data.data.name,
-          lastName: data.data.lastName,
-          phone: data.data.phone,
-          email: data.data.email,
-          password: data.data.password
-        });
-      }
-    } catch (error) {
-      alert(error);
+  useEffect(() => {
+    if (employees.length > 0 && id) {
+      console.log('entra');
+      const currentEmployee = employees.find((employee) => employee._id === id);
+      setIsEditing(true);
+      setEmployeeInput({
+        name: currentEmployee.name,
+        lastName: currentEmployee.lastName,
+        phone: currentEmployee.phone,
+        email: currentEmployee.email,
+        password: currentEmployee.password
+      });
     }
   }, []);
```

Comment:
you have to add both dependencies in the array because you're comparing both things
```suggestion
  }, [employees.length, id]);
```